Resizetizer.NT/Resizetizer.NT.targets

```diff
@@ -159,8 +201,110 @@
+		</DetectInvalidResourceOutputFilenamesTask>
+
+		<!-- Copy font files over -->
+		<Copy
+			SourceFiles="@(SharedFont)"
+			DestinationFolder="$(IntermediateOutputPath)sp\fonts\"
+			SkipUnchangedFiles="true" />
+
+
+		<ItemGroup>
+			<_SharedFontCopied Include="$(IntermediateOutputPath)sp\fonts\*" />
+		</ItemGroup>
+
+		<ItemGroup Condition="'$(_ResizetizerIsiOSApp)' == 'True' And '@(_SharedFontCopied)' != ''">
+
+			<!-- iOS Expects images in this group -->
+			<_SharedFontBundleResource Include="@(_SharedFontCopied)">
+				<LogicalName>$([System.IO.Path]::GetFileName(%(_SharedFontCopied.Identity)))</LogicalName>
+				<TargetPath>$([System.IO.Path]::GetFileName(%(_SharedFontCopied.Identity)))</TargetPath>
+			</_SharedFontBundleResource>
+
+			<BundleResource Include="@(_SharedFontBundleResource)" />
+
+		</ItemGroup>
+		
+		<!-- Create a partial info.plist for iOS -->
+		<CreatePartialInfoPlistTask
+			Condition="'$(_ResizetizerIsiOSApp)' == 'True' And '@(_SharedFontCopied)' != ''"
+			IntermediateOutputPath="$(IntermediateOutputPath)sp\"
+			PlistName="FontInfo.plist"
+			CustomFonts="@(_SharedFontCopied)">
+			<Output
+				TaskParameter="PListFiles"
+				ItemName="_SharedFontPListFiles" />
+				
+		</CreatePartialInfoPlistTask>
+
+		<!-- iOS - Partial Info.plist for font registration  -->
+		<ItemGroup Condition="'$(_ResizetizerIsiOSApp)' == 'True' ">
+			<_PartialAppManifest Include="@(_SharedFontPListFiles)" Condition="'@(_SharedFontPListFiles)' != ''" />
+			<FileWrites Include="@(_SharedFontPListFiles)" Condition="'@(_SharedFontPListFiles)' != ''" />
+		</ItemGroup>
+
+		<!-- Android -->
+		<ItemGroup Condition="'$(_ResizetizerIsAndroidApp)' == 'True'">
+
+			<AndroidAsset Include="@(_SharedFontCopied)" Condition="'@(_SharedFontCopied)' != ''">
+				<Link>$([System.IO.Path]::GetFileName(%(_SharedFontCopied.Identity)))</Link>
+			</AndroidAsset>
+
+		</ItemGroup>
+
+		<!-- WPF -->
+		<ItemGroup Condition="'$(_ResizetizerIsWPFApp)' == 'True'">
+
+			<Resource Include="@(_SharedFontCopied)" Condition="'@(_SharedFontCopied)' != ''">
+				<Link>$([System.IO.Path]::GetFileName(%(_SharedFontCopied.Identity)))</Link>
+			</Resource>
+
+		</ItemGroup>
+
+		<!-- UWP -->
+		<ItemGroup Condition="'$(_ResizetizerIsUWPApp)' == 'True'">
+
+			<ContentWithTargetPath Include="@(_SharedFontCopied)" Condition="'@(_SharedFontCopied)' != ''">
+				<TargetPath>Assets\$([System.IO.Path]::GetFileName(%(_SharedFontCopied.Identity)))</TargetPath>
+			</ContentWithTargetPath>
+
+		</ItemGroup>
+
+		<!-- iOS Only -->
+		<!-- If on Windows, using build host, copy the files over to build server host too -->
+		<CopyFilesToBuildServer
+			Condition="'$(BuildSessionId)' != '' And '$(_ResizetizerIsiOSApp)' == 'True' And '$(IsMacEnabled)'=='true'"
+			SessionId="$(BuildSessionId)"
+			Files="@(_SharedFontBundleResource)" />
```

Comment:
Will this copy the partial manifest as well? It doesn't seem to be in that group.

Reply:
Good catch, fixed now.